### PR TITLE
(no ticket): [update] Node Sass Sunset, CTA

### PR DESCRIPTION
**Pull request by jamie-mckenzie**
# (no ticket): [update] Node Sass Sunset, CTA

## What changed?

* Update call to action.

## Release notes draft

* 

## Anything else?

ping @bc-traciporter @slsriehl

### CODE REVIEW

**Comment by bc-traciporter** [comment]
@jamie-mckenzie Is it okay to keep the January 31 date in Scenario 1?
"Please review and publish anytime prior to January 31, 2024."

File: docs/stencil-docs/installing-stencil-cli/node-sass.mdx [hunk 1]
```diff
@@ -56,4 +56,4 @@ stencil push
 
 **How long do I have? When will the legacy SCSS compiler (node-sass) be sunset?**
 
-Please make these changes immediately; we are sunsetting support for the `node-sass` package on **January 31, 2024**. After this date, you must compile all SCSS files with Stencil's updated SCSS compiler engine. This upgrade is necessary to ensure the security of our platform and to ensure that you are not running out-of-date packages that may expose your projects to additional security vulnerabilities.
+Please make these changes as soon as possible; we are sunsetting support for the `node-sass` package on **January 31, 2024**. After this date, your current production site will continue to function as designed, but all future updates will require you to address any SCSS compatibility issues with node.js 18. This upgrade is necessary to ensure the security of our platform and to ensure that you are not running out-of-date packages that may expose your projects to additional security vulnerabilities. Thank you.
```
Comment (slsriehl) [inline docs/stencil-docs/installing-stencil-cli/node-sass.mdx:59]:
```suggestion
Please make these changes as soon as possible; we are sunsetting support for the `node-sass` package on **January 31, 2024**. After this date, your current production site will continue to function as designed, but all future updates will require you to address any SCSS compatibility issues with Node.js 18. This upgrade is necessary to ensure the security of our platform and to ensure that you are not running out-of-date packages that may expose your projects to additional security vulnerabilities.
```

**Review by bc-traciporter — COMMENTED** [review]
left a comment

**Review by bc-tgomez — APPROVED** [review]
Thanks!

**Pull request merged**